fs/types.go

```diff
@@ -0,0 +1,69 @@
+package fs
+
+import "time"
+
+type DeviceInfo struct {
+	Device string
+	Major  uint
+	Minor  uint
+}
+
+type FsType string
+
+func (ft FsType) String() string {
+	return string(ft)
+}
+
+const (
```

Comment:
Those cons have to be exported and they are commented by itself (by name), so comments would just duplicate names of these.